afdd_stack/scripts/bootstrap.sh

```
@@ -283,10 +289,32 @@ print(f"Wrote {cfg_path} (web_root -> {dist})")
+  local cfg="$1"
+  [[ -f "$cfg" ]] || return 0
+  if [[ "${OFDD_VOLTTRON_CONFIG_STRICT:-0}" == 1 ]]; then
+    return 0
+  fi
+  local cert key
+  cert=$(grep -cE '^[[:space:]]*web-ssl-cert[[:space:]]*=' "$cfg" 2>/dev/null || true)
+  key=$(grep -cE '^[[:space:]]*web-ssl-key[[:space:]]*=' "$cfg" 2>/dev/null || true)
+  cert=${cert:-0}
+  key=${key:-0}
+  if [[ "$cert" -gt 1 ]] || [[ "$key" -gt 1 ]]; then
+    local bak="${cfg}.bak.duplicate-ssl.$(date +%s)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)